Directory.Build.props

```diff
@@ -1,5 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
+    <Version>1.1.0</Version>
```

Comment:
We are already on this version, or we are targeting it? If it is the later, I would advise not to put it there without any clear sign of "in development" status.

Reply:
I can definitely do that with version prefix and suffix, but we would have to change it every time before release. Currently it pulls these states from configs - making these suffixes appear automatically in development and hide in release. I agree that it's not clearly marked in code if someone looks in repository - on the other side, if someone looks for it, they will look first into "Releases" anyway (Getting exact commit of release). This si just to make our job easier 🐰

Reply:
It would be preferable if increasing a patch/minor/major version would be done with the actual release.
The current version is 1.0.1.